ansible/roles/postgresql-patroni-ansible-role/inventory/hosts

```diff
@@ -0,0 +1,10 @@
+[etcd]
```

Comment:
This file is not required. Please move this information on how to define the inventory into the `Readme.md` file